src/log.c

```diff
@@ -648,8 +685,12 @@ qd_error_t qd_log_entity(qd_entity_t *entity)
             src->includeSource = include_source;
         }
 
+        bool add_log_stats = src->add_log_stats;
+        src->add_log_stats = false;
         sys_mutex_unlock(log_source_lock);
-
+        if(add_log_stats) {
+            qd_entity_cache_add(QD_LOG_STATS_TYPE, src);
```

Comment:
I am unsure why we need to add the QD_LOG_STATS_TYPE here ?
I commented this line and the logStats was still created because qd_log_source() is creating it.